src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -8,6 +8,12 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Semantics
 {
     public class RecordTests : CompilingTestBase
     {
+        private static CSharpCompilation CreateCompilation(CSharpTestSource source)
```

Comment:
I thought we were moving away from these types of helpers?

Reply:
Not to my knowledge. Why? The plus side of it is that you can remove the helper after we ship and stuff will "just work"

Reply:
Checked with Jared, it's only about creating overloads with weird argument reordering or unnecessary custom target framework stuff.